src/components/Position/PositionAfter.tsx

```diff
@@ -33,13 +35,13 @@ export const PositionAfter: FC<PositionAfterProps> = ({
 }) => {
   const baseTokenAmountFormatted = useMemo(
     () =>
-      DecimalFormat.format(displayCollateralToken ?? Decimal.ZERO, {
+      DecimalFormat.format(displayCollateralTokenAmount ?? Decimal.ZERO, {
         style: 'currency',
-        currency: DISPLAY_BASE_TOKEN,
+        currency: TOKEN_TO_DISPLAY_BASE_TOKEN_MAP[displayCollateralToken],
```

Comment:
Prop `displayCollateralToken` is already a display token, no need to use `TOKEN_TO_DISPLAY_BASE_TOKEN_MAP` here I think.

Reply:
fixed